.circleci/config.yml

```diff
@@ -59,6 +58,58 @@ defaults_configure_git: &defaults_configure_git
     git config user.email ${GIT_CI_EMAIL}
     git config user.name ${GIT_CI_USER}
 
+defaults_configure_nvm: &defaults_configure_nvm
```

Comment:
I see you merged defaults_configure_nvm: &defaults_configure_nvm and defaults_configure_nvmrc: &defaults_configure_nvmrc from the sdk-scheme-adapter together. 

I'm fine with either grouping but just a reminder that we'll need to keep the structure in sync.

Reply:
@kleyow Here we're only interested in `defaults_configure_nvm: &defaults_configure_nvm`.

Reply:
Ah my mistake.